WordPress/Classes/ViewRelated/Reader/ReaderCommentsViewController.m

```diff
@@ -563,6 +566,18 @@ - (BOOL)canComment
     return self.post.commentsOpen && self.isLoggedIn;
 }
 
+- (BOOL)canFollowPost
+{
+    // FIXME: Older a8c internal P2s do not contain the isWPForTeams flag.
+    // In case we can't find a flag that marks an old P2 site as being a P2,
+    // we can assume that blogs in the Reader's Automattic tab are P2s.
+    // Note that blogs in the Reader's Automattic tab are only available to Automatticians.
+    NSManagedObjectContext *context = [[ContextManager sharedInstance] mainContext];
```

Comment:
Another logic that we can extract from the VC. :)

Reply:
Thinking of extracting to the [FollowCommentsService](https://github.com/wordpress-mobile/WordPress-iOS/pull/14655#discussion_r472122291)!

Reply:
Fixed! 65e25c0